src/pages/filter/ui/query-builder/groups/query-builder-groups.tsx

```diff
@@ -73,7 +73,7 @@ export const QueryBuilderGroups = observer((): ReactElement => {
 
         <div
           className="overflow-y-auto"
```

Comment:
This fix affects "Decision Tree".  The bottom chart is not fully loaded

Before fix:
![image](https://user-images.githubusercontent.com/87524295/156177504-2b73849f-47e0-4b55-aa78-703e327c36ac.png)

After fix:
![image](https://user-images.githubusercontent.com/87524295/156177999-f7f5b714-ae87-41c1-a500-4bbf16a7e3f3.png)